### ISSUE TITLE
Request sending does not complete when globals object is missing

### ISSUE THREAD

**Issue by MJGTwo** [a85/POSTMan-Chrome-Extension#5354]
**Describe the bug**
Regardless of the IP, Postman hangs on all requests. I have deleted the app and reinstalled it on my Mac and still no dice.

**To Reproduce**
1. Open Postman
2. type in `http://echo.getpostman.com/get` into the request URL
3. Set type to GET
4. Click Submit
5. Hangs 

**Expected behavior**
I expect the app to successfully connect with http://echo.getpostman.com/get. I have visited the webpage with my browser and I successfully download a file.

**Screenshots**
[Example](https://imgur.com/a/T62R5fX
)
**App information (please complete the following information):**
 - App Type: Native App
 - Postman Version: 6.4.2
 - OS: macOS Mojave 10.14

**Additional context**
I'm pretty sure the issue was me updating my OpenSSL version by compiling from source, which was successfully added to /usr/local/bin/openSSL. It's on version-1.1.2. I removed it and revert back to the system copy, which is LibreSSL 2.6.4. I did not have this issue until this. 

Additionally, I have the same issue when trying to connect to an HTTP server on my local machine at localhost:8080, which I can connect to via browser.

**Comment by MJGTwo** [comment]
I found the main.log file and received this error when I press Send:


`[10475][1539378842771][main][error]["Main~handleUncaughtError - Uncaught errors",
{
"name":"TypeError",
"message":"Cannot read property 'environment' of undefined",
"stack":"TypeError: Cannot read property 'environment' of undefined\n    at Object.startRun (/Applications/Postman.app/Contents/Resources/app/services/RuntimeExecutionService.js:143:14)\n    at EventEmitter.<anonymous> (/Applications/Postman.app/Contents/Resources/app/services/RuntimeExecutionService.js:421:33)\n    at emitOne (events.js:115:13)\n    at EventEmitter.emit (events.js:210:7)\n    at _dispatch (/Applications/Postman.app/Contents/Resources/app/common/_event-bus.js:180:14)\n    at EventEmitter.<anonymous> (/Applications/Postman.app/Contents/Resources/app/common/_event-bus.js:196:11)\n    at emitThree (events.js:135:13)\n    at EventEmitter.emit (events.js:216:7)\n    at WebContents.<anonymous> (/Applications/Postman.app/Contents/Resources/electron.asar/browser/api/web-contents.js:266:13)\n    at emitTwo (events.js:125:13)\n    at WebContents.emit (events.js:213:7)"
}]`

**Comment by MJGTwo** [comment]
Found out about `NODE_DEBUG=request` so I ran `NODE_DEBUG=request /Applications/Postman.app/Contents/MacOS/Postman` and got the following output (same thing basically)

```
Main~handleUncaughtError - Uncaught errors TypeError: Cannot read property 'environment' of undefined
    at Object.startRun (/Applications/Postman.app/Contents/Resources/app/services/RuntimeExecutionService.js:143:14)
    at EventEmitter.<anonymous> (/Applications/Postman.app/Contents/Resources/app/services/RuntimeExecutionService.js:421:33)
    at emitOne (events.js:115:13)
    at EventEmitter.emit (events.js:210:7)
    at _dispatch (/Applications/Postman.app/Contents/Resources/app/common/_event-bus.js:180:14)
    at EventEmitter.<anonymous> (/Applications/Postman.app/Contents/Resources/app/common/_event-bus.js:196:11)
    at emitThree (events.js:135:13)
    at EventEmitter.emit (events.js:216:7)
    at WebContents.<anonymous> (/Applications/Postman.app/Contents/Resources/electron.asar/browser/api/web-contents.js:266:13)
    at emitTwo (events.js:125:13)
    at WebContents.emit (events.js:213:7)
1539463376477 main error ["Main~handleUncaughtError - Uncaught errors",{"name":"TypeError","message":"Cannot read property 'environment' of undefined","stack":"TypeError: Cannot read property 'environment' of undefined\n    at Object.startRun (/Applications/Postman.app/Contents/Resources/app/services/RuntimeExecutionService.js:143:14)\n    at EventEmitter.<anonymous> (/Applications/Postman.app/Contents/Resources/app/services/RuntimeExecutionService.js:421:33)\n    at emitOne (events.js:115:13)\n    at EventEmitter.emit (events.js:210:7)\n    at _dispatch (/Applications/Postman.app/Contents/Resources/app/common/_event-bus.js:180:14)\n    at EventEmitter.<anonymous> (/Applications/Postman.app/Contents/Resources/app/common/_event-bus.js:196:11)\n    at emitThree (events.js:135:13)\n    at EventEmitter.emit (events.js:216:7)\n    at WebContents.<anonymous> (/Applications/Postman.app/Contents/Resources/electron.asar/browser/api/web-contents.js:266:13)\n    at emitTwo (events.js:125:13)\n    at WebContents.emit (events.js:213:7)"}]
```

Basically saying that the parameter `variable` of `Object.runtime` is undefined, which corrosponds to the call using the variable `message.data.variables`

**Comment by MJGTwo** [comment]
I found that `~/Library/Application Support/Postman` stayed around between uninstalls. I deleted it, uninstalled Postman, and reinstalled it. This fixed my issue. Closing.

**Comment by kamalaknn** [comment]
Hi @MJGTwo 

I walked through the code and found a possibility where this error might have happened.

It looks like there is no `globals` container for your workspace. Even if you do not have any values in your `globals` the container should always exist.

I have fixed the code flow while sending the request to continue working even if `globals` container is missing. We'll push it out in the next release.

We are still not sure why the `globals` container is missing. We're you signed in when this happened? Can you give us any debugging info that might help us with that?

**Comment by MJGTwo** [comment]
Hi @kamalaknn 

I was signed in with my Google account. Every time I reinstalled the app, I would promptly log back into my account. The last time I uninstalled, I did a `find` search via my terminal and found the config folder in `~/Library/Application Support/Postman` as stated above and deleted it entirely. Consequently, this also removed all logs from Postman during the time that I had this issue, so I cannot give you any debug information.
